api/app/signals/apps/api/v1/serializers/nested/status.py

```diff
@@ -39,17 +46,26 @@ def validate(self, attrs):
 
 
 class _NestedPublicStatusModelSerializer(serializers.ModelSerializer):
-    state_display = serializers.CharField(source='get_state_display', read_only=True)
+    state_display = serializers.SerializerMethodField(method_name='get_public_state_display')
+    state = serializers.SerializerMethodField(method_name='get_public_state')
 
     class Meta:
         model = Status
         fields = (
-            'id',
             'state',
             'state_display',
         )
         read_only_fields = (
-            'id',
             'state',
             'state_display',
```

Comment:
Not needed anymore, a SerializerMethodField is by default read-only

Reply:
We should not expose `id` of status anyway. I wonder whether we can remove it altogether. (There is no way for clients to get from us the statusses by id.)